server/routes/companies.js

```diff
@@ -24,7 +24,7 @@ read.get("/get/companies/:query", getCompanies);
 //Update a specific company
 // ! TEMP READ COMMAND
 // TODO: change up backend to remove client changing of companies
-read.patch("/update/:name", bodyParser.json(), updateCompany);
+read.patch("/update/:id", bodyParser.json(), updateCompany);
```

Comment:
I see we're fetching by id now which is great, did we change the `companyController.js` so it handles this?

Reply:
Oh nevermind, I see you changed the company backend, it just didn't render it cause it was so long.